docs/Reference/Routes.md

```diff
@@ -653,3 +653,7 @@ fastify.route({
   }
 })
 ```
+
+### ⚠  HTTP version check
+
+Fastify will check the HTTP version of every request and, based on [configuration options](./Server.md#http2), will check against one or all of the following versions: `2.0`, `1.1`, and `1.0`. If Fastify receives a different HTTP version in the request it will return a `505 HTTP Version Not Supported` error.
```

Comment:
Please hard wrap these lines to be consistent with the rest of the document. Also:
```suggestion
Fastify will check the HTTP version of every request, based on [configuration options](./Server.md#http2), to determine if it matches one or all of the following versions: `2.0`, `1.1`, and `1.0`. If Fastify receives a different HTTP version in the request it will return a `505 HTTP Version Not Supported` error.
```